client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -205,8 +207,18 @@ class SiteIconSetting extends Component {
 					<InfoPopover position="bottom right">
 						{ translate(
 							'The Site Icon is used as a browser and app icon for your site.' +
-								' Icons must be square, and at least %s pixels wide and tall.',
-							{ args: [ 512 ] }
+								' Icons must be square, and at least %s pixels wide and tall.' +
+								' {{a}}Learn more{{/a}}',
+							{
+								args: [ 512 ],
+								components: {
+									a: (
+										<InlineSupportLink
+											supportLink={ localizeUrl( 'https://wordpress.com/support/site-icons/' ) }
```

Comment:
Would it be best to add the link here so that it can be reused and so that users don't need to leave Calypso when reading the document? https://github.com/Automattic/wp-calypso/blob/trunk/client/components/inline-support-link/context-links.js

Reply:
> The InlineSuportLink component is designed to open the link in the new tab. It's not an optional parameter.

If you provide the support document link and ID within `context-links.js`, and then use the `supportContext` prop, it should load the article within the Help Centre. That's what seems to happen with the rest of the Settings support documentation. :) 

<img width="1039" alt="Screenshot 2024-09-17 at 13 32 58" src="https://github.com/user-attachments/assets/d791e4c5-9788-4c6b-819c-ea9890a1526d">

Reply:
Ough, I see. I forgot about this feature.
I'll try doing it that way.

Thanks @Aurorum!